packages/account/src/Components/poi/status/unsupported/documents.tsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { Card } from '@deriv/components';
 import { CardWithArrow } from './card-with-arrow';
-import { FormikValues } from 'formik';
+import { getDocumentIndex } from './constants';
 
 type TDocument = {
     toggleDetail: (index: number) => void;
-    documents: object[];
+    documents: ReturnType<typeof getDocumentIndex>;
 };
 
 export const Documents = ({ toggleDetail, documents }: TDocument) => (
     <React.Fragment>
-        {documents.map((item: FormikValues, index: number) => (
+        {documents.map((item, index: number) => (
             <Card
                 style={{
                     margin: '1.6rem 0',
                     height: '11.2rem',
                 }}
-                key={index}
+                key={item.card.icon}
```

Comment:
what if item.card is undefined? can we use optional chaning here?

Reply:
@amina-deriv, all these are hardcoded [here](https://github.com/binary-com/deriv-app/blob/master/packages/account/src/Components/poi/status/unsupported/constants.ts), so the `icon` is always defined but I'll add the optional chaining for future proof I guess